doc/level1.rst

```diff
@@ -140,11 +141,11 @@ attributes:
    This SHOULD default to ``nopreference``.
 
 If Autocrypt is enabled for a given account, the MUA SHOULD allow the
-user to switch the setting for ``own_state.prefer_encrypt``.  This
+user to switch the setting for ``accounts[A].prefer_encrypt``.  This
 choice might be hidden in something like a "preferences pane".  See
 :ref:`preference-ui` for a specific example of how this could look.
 
-How this information is managed and used is discussed in :ref:`own-state-management`.
+How this information is managed and used is discussed in :ref:`accounts-management`.
 
 .. _peer-state-management:
```

Comment:
Should peer-state-management also be renamed to peer-management?

Reply:
are you just asking about the anchor label, or the visible text?

if it's the anchor label, it should be ``peers-management`` for symmetry with ``accounts-management``.

If you're asking about the visible text, i'm uneasy with "peer management" because it could be read as implying that you're managing your peers, when what you're managing is state about those peers.

Reply:
peers-management is awkward.  I'm using peer-management instead and I changed accounts-management to account-management.